contrib/misc/bareos_pam_integration/README.rst

```diff
@@ -126,7 +133,29 @@ You may want to add following section to your :file:`/etc/bareos-webui/directors
    pam_console_name     = "pam-webui"
    pam_console_password = "secret"
 
-Now you should be able to login to the WebUI using PAM users.
+PAM users require a dedicated User Resource, see https://docs.bareos.org/master/Configuration/Director.html#user-resource .
+
+A working example User Resource configuration for a WebUI user named Alice
+looks like following.
+
+:file:`/etc/bareos/bareos-dir.d/user/Alice.conf`
+
+::
+
+   User {
+      Name = "Alice"
+      CommandACL = !.bvfs_clear_cache, !.exit, !.sql, !configure, !create, !delete, !purge, !prune, !sqlquery, !umount, !unmount, *all*
+      Job ACL = *all*
+      Schedule ACL = *all*
+      Catalog ACL = *all*
+      Pool ACL = *all*
+      Storage ACL = *all*
+      Client ACL = *all*
+      FileSet ACL = *all*
+      Where ACL = *all*
+   }
```

Comment:
As this section is not about User resources, I would prefer a short resource like:
```
User {
  Name = "alice"
  Profile = "webui-admin"
}
```
This also would correspond to https://docs.bareos.org/master/IntroductionAndTutorial/InstallingBareosWebui.html#create-a-restricted-consoles

Reply:
Changed